Automattic/woocommerce-payments
# Move TypeScript definitions for AccountOverview to sharable location

Fixes #5772

#### Changes proposed in this Pull Request









This PR moves TypeScript definitions for the `AccountOverview` namespace from the [`DepositsInformation`](https://github.com/Automattic/woocommerce-payments/blob/4ec527e374f2b72fc8f86b4a3fed43b2428e0193/client/components/deposits-information/declarations.d.ts) component directory to `client/types/account-overview.d.ts`, suitable for shared use across the `client`  dir.

This is necessary since the DepositsInformation component will be removed once the Simplify deposits UI project has concluded.

All references to the `AccountOverview` namespace will continue to reference this global namespace.

The unrelated `@wordpress/components` `Link` type has been moved to the existing `@woocommerce/components` module declaration in `client/deposits/declarations.d.ts`.


#### Testing instructions





* Run the following command to compile TypeScript files and ensure there are no errors:  
  `npx tsc --noEmit`


-------------------


**Post merge**
